Directory.Packages.props

```diff
@@ -24,7 +24,7 @@
     <PackageVersion Include="Azure.Security.KeyVault.Secrets" Version="4.7.0" />
     <PackageVersion Include="Azure.Storage.Blobs" Version="12.23.0" />
     <PackageVersion Include="Azure.Storage.Queues" Version="12.21.0" />
-    <PackageVersion Include="Microsoft.AspNetCore.SignalR.Client" Version="8.0.12" />
+    <PackageVersion Include="Microsoft.AspNetCore.SignalR.Client" Version="9.0.1" />
```

Comment:
I think we want to stay on 8 for this. It is only used in tests.

Maybe since it is a dotnet/aspnetcore package, it should follow the same as the other aspnet packages.